Makefile

```diff
@@ -18,6 +21,10 @@ check: libtree
 		$(MAKE) -C $$dir check; \
 	done
 
+install: all
+	mkdir -p $(DESTDIR)$(BINDIR)
+	cp libtree $(DESTDIR)$(BINDIR)
```

Comment:
`cp` or `install`?

Reply:
can you add `.PHONY` too for non-file-targets?

Reply:
Good idea, I added a `.PHONY` target.

I used `cp` because that is POSIX complaint and will work for other unix systems such as solaris which do not have GNU `install`.